content/en/platform/corda/5.0-dev-preview-2/getting-started/first-flow.md

```diff
@@ -244,11 +244,11 @@ In the responder flow:
 ```
 ## Injecting Services
 Corda 5 requires a CorDapp Developer to explicitly specify which Corda services are required by the flow. In this simple example we use three services:
-* JsonMarshallingService — a service that CorDapps and other services may use to marshal arbitrary content in and out of JSON format using standard approved mappers.
-* FlowMessaging  — a service that CorDapps can use to create communication sessions between two virtual nodes. Once set up, you can send and receive using the session object.
-* MemberLookup  — a service that CorDapps can use to retrieve information about virtual nodes on the network.
+* `JsonMarshallingService` — a service that CorDapps and other services may use to marshal arbitrary content in and out of JSON format using standard approved mappers.
+* `FlowMessaging`  — a service that CorDapps can use to create communication sessions between two virtual nodes. Once set up, you can send and receive using the session object.
+* `MemberLookup`  — a service that CorDapps can use to retrieve information about virtual nodes on the network.
 
-There are other services, such as the Persistence and Serialization services, which are beyond the scope of this getting started example.
+There are other services, such as the `Persistence` and `Serialization` services, which are beyond the scope of this getting started example.
```

Comment:
```suggestion
There are other services, such as the `Persistence` and `Serialization` services, which are beyond the scope of this Getting Started example.
```

I'm putting caps for this but still uncertain they should be used so I'll leave it up to you!